apps/api/src/app/portfolio/portfolio.controller.ts

```diff
@@ -319,6 +320,28 @@ export class PortfolioController {
     return { holdings: Object.values(holdings) };
   }
 
+  @Get('lookup')
+  @UseGuards(AuthGuard('jwt'), HasPermissionGuard)
+  @UseInterceptors(TransformDataSourceInResponseInterceptor)
+  public async lookupSymbol(
+    @Headers(HEADER_KEY_IMPERSONATION.toLowerCase()) impersonationId: string,
+    @Query('accounts') filterByAccounts?: string,
```

Comment:
Not implemented on the client yet, but this allows us to filter by account if the user already selected an account.